groups/groups.yaml

```diff
@@ -11,26 +11,45 @@ groups:
       MessageModerationLevel: "MODERATE_NON_MEMBERS"
       ReconcileMembers: "true"
     owners:
-      - dprotaso@gmail.com
+      # SC
+      - aliok@redhat.com
+      - developer@nrrso.com
       - evan.k.anderson@gmail.com
-      - evana@vmware.com
-      - evankanderson@knative.team
+      - naina.sng@gmail.com
+      - salaboy@gmail.com
+      # TOC
+      - david.hadas@gmail.com
+      - dprotaso@gmail.com
+      - dsimansk@redhat.com
+      - d.simansky@gmail.com
       - paul@paulschweigert.com
       - paulschw@us.ibm.com
```

Comment:
Would this work?

```
owners:
      - toc@knative.team
      - steering@knative.team
```

Reply:
It won't, a group can't own another group. Its a limitation of the Google Groups API